packages/dds/pact-map/src/pactMap.ts

```diff
@@ -404,9 +421,7 @@ export class PactMap<T = unknown> extends SharedObject<IPactMapEvents> implement
 			return (
 				// Items have an effect if they are still pending, have a real value, or some client may try to
 				// reference state before the value was accepted.  Otherwise they can be dropped.
-				pact.pending !== undefined ||
-				pact.accepted.value !== undefined ||
-				pact.accepted.sequenceNumber > this.runtime.deltaManager.minimumSequenceNumber
+				pact.pending !== undefined || pact.accepted !== undefined
```

Comment:
@ChumpChief, does this seem correct? My understanding is we want to avoid summarizing entries that are neither pending/accepted. So even if a value was deleted (undefined), then `pact.accepted` should still be defined since it will have a sequence number defined and look something like:
```ts
{
  "value": undefined,
  "acceptedSequenceNumber": 14,
}
```

Do we still need to verify that `pact.accepted.sequenceNumber > this.runtime.deltaManager.minimumSequenceNumber`? To me this check seems unnecessary and was only required since we were checking `pact.accepted.value` in the past.

Reply:
Yep this is unnecessary, should always be true for each entry since entries are guaranteed to have at least one of pending/accepted defined.  So the whole filter statement can go.